lib/TokenClientBuilder.php

```diff
@@ -153,7 +153,13 @@ public function developerKey($devKey)
      */
     public function build()
     {
-        $composerJson = json_decode(file_get_contents(Factory::getComposerFile()), true);
+        $composerFilePath = Factory::getComposerFile();
```

Comment:
It seems we should just always use `../composer.json`? In case the `COMPOSER` is set on the host machine and points to something else.

Reply:
done